jekyll/_docs/enterprise/docker-install.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: enterprise
+title: "Installing CircleCI"
```

Comment:
Feels like this should be "non-AWS" or something, because as-is this will look like:

Installation
- AWS
- Installing CircleCI

Which seems weird.